builtbymagellan.com.websitehosting.json

```diff
@@ -0,0 +1,30 @@
+{
+  "providerId": "builtbymagellan.com",
+  "providerName": "Built by Magellan",
+  "serviceId": "websitehosting",
+  "serviceName": "Hébergement Site Web IA",
+  "version": 1,
+  "logoUrl": "https://builtbymagellan.com/logo.svg",
+  "description": "Connecte automatiquement votre domaine personnalisé à votre site web généré par IA",
+  "syncBlock": false,
+  "syncRedirectDomain": "builtbymagellan.com",
+  "warnPhishing": false,
+  "hostRequired": false,
+  "records": [
+    {
+      "type": "APEXCNAME",
```

Comment:
I hope you are aware this is an extension feature therefore not widely supported.